Source/Libraries/openXDA.APIMiddleware/APIPingController.cs

```diff
@@ -0,0 +1,54 @@
+﻿//******************************************************************************************************
+//  APIPingController.cs - Gbtc
+//
+//  Copyright © 2023, Grid Protection Alliance.  All Rights Reserved.
+//
+//  Licensed to the Grid Protection Alliance (GPA) under one or more contributor license agreements. See
+//  the NOTICE file distributed with this work for additional information regarding copyright ownership.
+//  The GPA licenses this file to you under the Eclipse Public License -v 1.0 (the "License"); you may
+//  not use this file except in compliance with the License. You may obtain a copy of the License at:
+//
+//      http://www.opensource.org/licenses/eclipse-1.0.php
+//
+//  Unless agreed to in writing, the subject software distributed under the License is distributed on an
+//  "AS-IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. Refer to the
+//  License for the specific language governing permissions and limitations.
+//
+//  Code Modification History:
+//  ----------------------------------------------------------------------------------------------------
+//  06/27/2023 - Christoph Lackner
+//       Generated original version of source code.
+//
+//******************************************************************************************************
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Security.Principal;
+using System.Web.Http;
+using GSF;
+using GSF.ServiceProcess;
+
+namespace openXDA.APIMiddleware
+{
+    /// <summary>
+    /// Base Class that includes the endpoint needed for Pinging the service.
+    /// </summary>
+    public abstract class  APIPingController : ApiController
+    {
+
+        #region [ HTTP Methods ]
+
+        /// <summary>
+        /// Returns a 1 if the service is running
+        /// </summary>
+        /// <returns> a simple 1 </returns>
+        [Route(""), HttpGet]
+        public IHttpActionResult Connect()
+        {
+            return Ok(1);          
+        }
+
+        #endregion
```

Comment:
We don't need a region here since there is only one method. Rather than helping to organize the code, it just adds extra clutter.
```suggestion
        /// <summary>
        /// Returns a 1 if the service is running
        /// </summary>
        /// <returns> a simple 1 </returns>
        [Route(""), HttpGet]
        public IHttpActionResult Connect()
        {
            return Ok(1);          
        }
```